tests/manage/z_cluster/cluster_expansion/test_add_non_ocs_workers.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import tier1, ManageTest
+from ocs_ci.ocs.node import add_nodes, taint_nodes, get_worker_nodes_not_in_ocs
+from ocs_ci.ocs.resources.pod import get_pod_node, get_plugin_pods
+
+logger = logging.getLogger(__name__)
+
+
+@tier1
+class TestAddNonOcsNodes(ManageTest):
```

Comment:
The main intent of the test case is to check toleration for Ceph CSI driver DS on non ocs nodes, please change the script title and class name according to it.

Reply:
done